assets/src/components/map.tsx

```diff
@@ -597,9 +682,24 @@ const Map = (props: Props): ReactElement<HTMLDivElement> => {
         {(props.trainVehicles || []).map((trainVehicle: TrainVehicle) => (
           <TrainVehicle key={trainVehicle.id} trainVehicle={trainVehicle} />
         ))}
+
         {(props.shapes || []).map((shape) => (
-          <Shape key={shape.id} shape={shape} />
+          <RouteShape key={shape.id} shape={shape} />
         ))}
+        {
+          <RouteStops
+            stops={(props.shapes || []).flatMap((shape) => shape.stops || [])}
+            iconSize={stationIconSize}
+          />
+        }
```

Comment:
**question:** does this need to be wrapped in curly braces if it's just a plain JSX element?